gc/verbose/VerboseWriterFileLogging.cpp

```diff
@@ -294,6 +294,12 @@ MM_VerboseWriterFileLogging::closeStream(MM_EnvironmentBase *env)
 	closeFile(env);
 }
 
+bool
+MM_VerboseWriterFileLogging::openStream(MM_EnvironmentBase *env)
+{
+	return openFile(env, true);
```

Comment:
think about adding a comment what's the meaning of this 'true' and why we pass it